src/__tests__/lib/containers/SynapseTable.test.tsx

```diff
@@ -606,27 +608,33 @@ describe('basic functionality', () => {
         },
         [mockUserCardValue]: {},
       }
+
+      const tableCellProps = {
+        entityColumnIndicies,
+        userColumnIndicies,
+        dateColumnIndicies,
+        dateListColumnIndicies,
+        booleanListColumnIndicies,
+        fileHandleIdColumnIndicies,
+        entityIdListColumnIndicies,
+        otherListColumnIndicies,
+        isBold: '',
+        mapUserIdToHeader: {},
+        columnLinkConfig: undefined,
+        rowIndex: undefined,
+        columnName: '',
+        selectColumns: undefined,
+        columnModels: undefined,
+      }
+
       it('renders an entity link', () => {
         const tableCell = shallow(
           <div>
             {renderTableCell({
-              entityColumnIndicies,
-              userColumnIndicies,
-              dateColumnIndicies,
-              booleanListColumnIndicies,
-              fileHandleIdColumnIndicies,
-              dateListColumnIndicies,
-              otherListColumnIndicies,
+              ...tableCellProps,
```

Comment:
The table cell props were duplicated in several places so I refactored it.

Reply:
Much cleaner, thank you.